src/Algebra/WithApartness/Structures.agda

```diff
@@ -0,0 +1,56 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Algebraic structures with an apartness relation
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra.Core using (Op₁; Op₂)
+open import Relation.Binary.Core using (Rel)
+
+module Algebra.WithApartness.Structures
+  {c ℓ₁ ℓ₂} {Carrier : Set c}
+  (_≈_ : Rel Carrier ℓ₁)
+  (_#_ : Rel Carrier ℓ₂)
+  (_+_ _*_ : Op₂ Carrier) (-_ : Op₁ Carrier) (0# 1# : Carrier)
```

Comment:
So the parameters on this line should be local to each definition. Otherwise when in future we add more structures which have a different set of operations, we'll have to break compatibility.

Reply:
done.